src/main/resources/site/pages/default/default.html

```diff
@@ -1,3 +1,4 @@
+<!DOCTYPE html>
```

Comment:
This could probably be removed. The code is duplicated when viewing page source on a page. Additionally, the w3c validator reports a "Stray doctype" error.